build/arm.sh

```diff
@@ -98,9 +98,19 @@ arm_unmount()
 	umount ${STAGEDIR}
 }
 
+arm_install_efi()
+{
+	DEV_EFI=$(mdconfig -a -t vnode -f ${STAGEDIR}/boot/boot1.efifat)
+	mount_msdosfs /dev/${DEV_EFI} ${STAGEDIR}/mnt
+	cp -r ${STAGEDIR}/mnt/efi ${STAGEDIR}/boot/msdos/efi
+	umount ${STAGEDIR}/mnt
+	mdconfig -d -u ${DEV_EFI}
+}
+
 echo -n ">>> Building arm image... "
 
 arm_install_uboot
+arm_install_efi
```

Comment:
There is an UEFI flag for this... https://github.com/opnsense/tools/blob/17fa565a1589d029169504b3414d9aea4532b193/build/vm.sh#L87-L88

https://github.com/opnsense/tools/blob/17fa565a1589d029169504b3414d9aea4532b193/Makefile#L84

I don't mind adding UEFI to arm by default. But it would be nice to be able to disable it.

Reply:
Okay, I have made EFI installing for arm default and optional.